spec/pivotal-ui-react/notification/notification_spec.js

```diff
@@ -14,18 +29,32 @@ describe('Notification', function() {
   describe('when there are children', function() {
     beforeEach(function() {
       React.render((
-        <Notifications>
-          <NotificationItem href="my-fee-link">fee</NotificationItem>
+        <Notifications {...props}>
+          <NotificationItem {...itemProps} href="my-fee-link">fee</NotificationItem>
           <NotificationItem href="my-fi-link">fi</NotificationItem>
           <NotificationItem href="my-fo-link">fo</NotificationItem>
           <NotificationItem href="my-fum-link">fum</NotificationItem>
         </Notifications>), root);
     });
 
+    it('passes through the className to the btn-group', function() {
+      expect('#root .btn-group').toHaveClass(props.className);
+    });
+
+    it('passes through style to the button', function() {
+      expect('#root .btn').toHaveCss(props.style);
+    });
+
+    it('passes through id to the button', function() {
+      expect('#root .btn#test-id').toExist();
+    });
+
```

Comment:
Given that id and style are not propagated in the standard way, we should probably document this.